### PR TITLE
StandardUI-Windows: Remove unused platforms.

### PR DESCRIPTION
**Pull request by srdjanjovcic**

### CODE REVIEW

**Comment by srdjanjovcic** [comment]
This removes Blazor projects from the configuration, and also removes platform-specific platforms from solution configuration.

File: StandardUI-Windows.sln [hunk 1]
```diff
@@ -31,172 +31,60 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "external", "external", "{7A
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.ComponentModelEx", "external\component-model-ex\src\Microsoft.ComponentModelEx\Microsoft.ComponentModelEx.csproj", "{A3FD54C6-4006-4B77-9EC7-A809DD54B307}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.StandardUI.Blazor", "src\Microsoft.StandardUI.Blazor\Microsoft.StandardUI.Blazor.csproj", "{620002CD-1FD2-4E08-8C89-AF58DCF899F4}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "StandardUI.VisualEnvironment.Blazor", "src\StandardUI.VisualEnvironment.Blazor\StandardUI.VisualEnvironment.Blazor.csproj", "{6BB67F42-F654-4159-968F-EF74730EC1C4}"
+EndProject
```
Comment (BretJohnson) [inline StandardUI-Windows.sln:37]: Did you mean to remove those rather than add them?

File: StandardUI-Windows.sln [hunk 2]
```diff
@@ -31,172 +31,60 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "external", "external", "{7A
...
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{752F4208-8318-4C57-9868-9921FF743AA9}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{752F4208-8318-4C57-9868-9921FF743AA9}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{752F4208-8318-4C57-9868-9921FF743AA9}.Debug|arm64.ActiveCfg = Debug|Any CPU
-		{752F4208-8318-4C57-9868-9921FF743AA9}.Debug|arm64.Build.0 = Debug|Any CPU
-		{752F4208-8318-4C57-9868-9921FF743AA9}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{752F4208-8318-4C57-9868-9921FF743AA9}.Debug|x64.Build.0 = Debug|Any CPU
-		{752F4208-8318-4C57-9868-9921FF743AA9}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{752F4208-8318-4C57-9868-9921FF743AA9}.Debug|x86.Build.0 = Debug|Any CPU
```
Comment (BretJohnson) [inline StandardUI-Windows.sln:53]: Do you by chance know how I may have (accidentally) added all those platform configurations? In any case, thanks for cleaning this up.

File: StandardUI-Windows.sln (hunk 1 above)
Comment (srdjanjovcic) [reply StandardUI-Windows.sln:37]: Nah, I want them in ... They are just unselected in solution configuration, so they are not built when solution is built.

File: StandardUI-Windows.sln (hunk 2 above)
Comment (srdjanjovcic) [reply StandardUI-Windows.sln:53]: Probably there was a native C++ project in the solution at some point?

**Pull request merged**